package.json

```diff
@@ -18,7 +18,7 @@
     "functional": "BROWSERS='chrome,firefox' TEST_FILES='test/functional/specs/**/*.js' npm run testcafe",
     "functional-sl": "TEST_FILES='test/functional/specs/**/*.js' npm run testcafe",
     "prebuild": "npm run clean && npm run format && npm run lint",
-    "build": "rollup -c",
+    "build": "rollup -c && rollup -c rollup.prod.config.js",
```

Comment:
What's your reasoning for building the minified version on `npm run build`? I was thinking instead you would change `npm run build:prod` to build both the unminified and minified versions.

Reply:
Good question I didn't think of it this way. I thought why not build both all the times? I'm good either way ;)